core/src/style/polygonStyle.cpp

```diff
@@ -18,33 +21,45 @@ void PolygonStyle::constructVertexLayout() {
         {"a_color", 4, GL_UNSIGNED_BYTE, true, 0},
         {"a_layer", 1, GL_FLOAT, false, 0}
     }));
-    
+
 }
 
 void PolygonStyle::constructShaderProgram() {
-    
+
     std::string vertShaderSrcStr = stringFromResource("polygon.vs");
     std::string fragShaderSrcStr = stringFromResource("polygon.fs");
-    
+
     m_shaderProgram = std::make_shared<ShaderProgram>();
     m_shaderProgram->setSourceStrings(fragShaderSrcStr, vertShaderSrcStr);
 }
 
-void PolygonStyle::buildPoint(Point& _point, StyleParams& _params, Properties& _props, VboMesh& _mesh) const {
+void* PolygonStyle::parseStyleParams(StyleParamMap& _styleParamMap) const {
+    StyleParams* params = new StyleParams();
```

Comment:
Ohh :|. I had this in, when I commented out the usage of m_styleParamCache in uber style class.
Will add this back in.